src/ui/src/components/core/layout/CoreTabs.vue

```diff
@@ -36,6 +43,14 @@ export default {
 		description,
 		category: "Layout",
 		allowedChildrenTypes: ["tab", "repeater"],
+		events: {
+			"wf-tab-change": {
+				desc: "Sent when the active tab changes.",
+				stub: tabChangeHandlerStub.trim(),
+				eventPayloadExample: "Tab Name",
+				bindable: true,
```

Comment:
You didn't implement the bindable mode. Bindable mode means the current tab can be synced to a state variable, which is not the case here.

You can simply remove it from now

```suggestion
```